README.md

```diff
@@ -1057,6 +1057,120 @@ z.string().array().length(5); // must contain 5 items exactly
 
 Unlike `.nonempty()` these methods do not change the inferred type.
 
+### `.uniq`
```

Comment:
Do you see any benefits of using a shortened name for `unique`? Instead of `.unique()` more clear/less distraction for developers in my opinion!

Reply:
I where just conforming to Lodash wording that use the key .uniq as I more often see the key .uniq in libs and because I'm more used to this keyword.
But that's not really a concern for me, it's more a question of personal preference I think so I will not try argue on this point and simply align the PR if necessary.

Reply:
I think the name is confusing. `uniq` in lodash transforms the input, while the suggested method here checks and "returns" the input as-is.

Reply:
Is the function name the last remaining piece to solve? These other tools/specifications use some variation on the word **unique**:
- ajv: `uniqueItemProperties`
- joi: `unique`
- json-schema: `uniqueItems`

Perhaps in the future, array coercion might remove duplicates. If that's the case, any mutability implied by the word **unique** might actually be desirable? Anyway, just my 2¢.

The word **distinct** also seems to be a good fit here.